apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/lib/getIncidentDescriptionReason.ts

```diff
@@ -0,0 +1,86 @@
+import { IntlShape } from 'react-intl'
+
+import {
+  IndictmentCountOffense,
+  Substance,
+  SubstanceMap,
+} from '@island.is/judicial-system/types'
+import { Offense } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { indictmentCount as strings } from '../IndictmentCount.strings'
+import { indictmentCountSubstanceEnum as substanceStrings } from '../IndictmentCountSubstanceEnum.strings'
+
+export const getIncidentDescriptionReason = (
```

Comment:
Same as before, just extracted into a new file do declutter the indictment count file